pkg/virt-api/rest/subresource.go

```diff
@@ -1299,6 +1299,16 @@ func (app *SubresourceAPIApp) VMRemoveVolumeRequestHandler(request *restful.Requ
 	app.removeVolumeRequestHandler(request, response, false)
 }
 
+// VMAddInterfaceRequestHandler handles the subresource for hot plugging a volume and disk.
```

Comment:
It will be nice if you do not add to this never-ending file.
Does it make sense to have a file dedicated to this new network hotplug?

Same goes for the test file.

Reply:
Done.